qa-tests-backend/src/test/groovy/ReconciliationTest.groovy

```diff
@@ -108,10 +108,6 @@ class ReconciliationTest extends BaseSpecification {
         when:
         "Get Sensor and counts"
 
-        // Verify initial reconciliation stats (from the reconciliation that must have happened
-        // whenever the sensor first connected).
-        verifyReconciliationStats(false)
```

Comment:
We can now simplify that function by removing the bool parameter altogether, since we only call it once.

Reply:
Thanks. Updated